src/HentaiHere/HentaiHereParser.ts

```
@@ -0,0 +1,239 @@
+import { Chapter, ChapterDetails, Tag, HomeSection, LanguageCode, Manga, MangaStatus, MangaTile, MangaUpdates, PagedResults, SearchRequest, TagSection } from "paperback-extensions-common";
+
+const HH_DOMAIN = 'https://hentaihere.com'
+
+export const parseMangaDetails = ($: CheerioStatic, mangaId: string): Manga => {
+
+  const titles = [];
+  const titleContext = $("a", $("h4", "div.bg-black")).first();
+  titles.push(decodeHTMLEntity(titleContext.text().replace($("span.mngType", titleContext).text(), "").trim()));
```

When either of the `.text()` calls here are `""` because there was a parsing error, do you want to continue processing and show something in the app which is incorrect? Or would you prefer to throw an error here and let the app know there was an issue. (A good throw text so the app logs are useful) 

You could convince me that either way is fine, merely wanted to bring it up